src/components/Input/InputSmall.tsx

```diff
@@ -23,23 +25,28 @@ function InputSmall(props: Props) {
     onChange,
     onKeyUp,
     placeholder,
+    suffixes,
     type,
     value,
   } = props;
 
   return (
-    <input
-      className={`${className} appearance-none rounded-md border border-panel-border bg-panel-input text-sm transition ease-in-out focus:shadow-none focus:outline-none focus:ring-2 focus:ring-inset focus:ring-panel-icon-action`}
-      id={id}
-      type={type}
-      placeholder={placeholder ?? ''}
-      value={value}
-      onChange={onChange}
-      onKeyUp={onKeyUp}
-      autoFocus={autoFocus}
-      disabled={disabled}
-      autoComplete={autoComplete ?? 'on'}
-    />
+    <>
+      <input
+        className={`${className} appearance-none rounded-md border border-panel-border bg-panel-input text-sm transition ease-in-out focus:shadow-none focus:outline-none focus:ring-2 focus:ring-inset focus:ring-panel-icon-action`}
+        id={id}
+        type={type}
+        placeholder={placeholder ?? ''}
+        value={value}
+        onChange={onChange}
+        onKeyUp={onKeyUp}
+        autoFocus={autoFocus}
+        disabled={disabled}
+        autoComplete={autoComplete ?? 'on'}
+      />
+
+      {suffixes && suffixes(props)}
```

Comment:
Why are the props being passed back to the suffixes function?
They are not being used there.

And the suffixes `<div>` can just be added to the component here and we can just pass the number to this component instead of the function

Reply:
I trying to aim for a more dynamic render for suffixes if other dev want to render another kind of component or info as a suffixes of the Input

Reply:
Then just pass on the suffix component itself, don't pass on a render function.